### ISSUE TITLE
Extern completions handle comments incorrectly

### ISSUE THREAD

**Issue by KAAtheWiseGit** [nushell/nushell#12274]
### Describe the bug

I was making an external completion and noticed, that all of the comments (which are interpreted as option documentation) are shifted by one.

### How to reproduce

The completion file is:

```Nushell
export extern img2sixel [
	# Files to display
	...imagefiles: path

	# Specify output file name (default: `stdout`)
	--outfile (-o): path
	# Generate a sixel image for 7bit terminals or printers (default)
	--7bit-mode (-7)
	# Generate a sixel image for 8bit terminals or printers
	--8bit-mode (-8)
]
```

The resulting help message:

```Nushell
~/project/nu-completion/src> source img2sixel.nu
~/project/nu-completion/src> help img2sixel
Usage:
  > img2sixel {flags} ...(imagefiles) 

Flags:
  -o, --outfile <Filepath> - Generate a sixel image for 7bit terminals or printers (default)
  -7, --7bit-mode - Generate a sixel image for 8bit terminals or printers
  -8, --8bit-mode - 

Parameters:
  ...imagefiles <path>: Specify output file name (default: `stdout`)
```

### Expected behavior

Handle comments correctly.

### Configuration

| key                | value                                                        |
| ------------------ | ------------------------------------------------------------ |
| version            | 0.90.1                                                       |
| branch             |                                                              |
| commit_hash        |                                                              |
| build_os           | linux-x86_64                                                 |
| build_target       | x86_64-alpine-linux-musl                                     |
| rust_version       | rustc 1.76.0 (07dca489a 2024-02-04) (Alpine Linux 1.76.0-r0) |
| cargo_version      | cargo 1.76.0                                                 |
| build_time         | 2024-02-14 02:25:04 +00:00                                   |
| build_rust_channel | release                                                      |
| allocator          | mimalloc                                                     |
| features           | default, sqlite, trash, which, zip                           |
| installed_plugins  |                                                              |

**Comment by KAAtheWiseGit** [comment]
I forgot the descriptions must be on the same line as the options and arguments.  I'd suggest allowing an alternative syntax, because otherwise the lines get uncomfortably long.

**Comment by fdncred** [comment]
Correct, the comments for the help have to be on the same line currently. I can see how allowing them to be on the line above may be helpful too.

**Comment by KAAtheWiseGit** [comment]
@fdncred one additional thing it could enable is multi-line comments.  Some utilities have multi-line option descriptions.
